commons-vfs2-examples/src/main/java/org/apache/commons/vfs2/libcheck/package.html

```diff
@@ -1,21 +0,0 @@
-<!--
-    Licensed to the Apache Software Foundation (ASF) under one or more
-    contributor license agreements.  See the NOTICE file distributed with
-    this work for additional information regarding copyright ownership.
-    The ASF licenses this file to You under the Apache License, Version 2.0
-    (the "License"); you may not use this file except in compliance with
-    the License.  You may obtain a copy of the License at
-
-         http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
--->
-<body>
-<p>Sample clients for commons-net.ftp and JSch libraries.</p>

```

Comment:
@SethFalco 
Why was this information removed?

Reply:
It wasn't removed, it's over here: https://github.com/apache/commons-vfs/pull/206/files#diff-0664cd55eac2920b414dd49f3b570762e284d76f2fd462c85eb46d870c81e4c1R19

Reply:
Gotcha! TY.